lib/improver/utilities/save.py

```diff
@@ -137,15 +138,31 @@ def save_netcdf(cubelist, filename):
             Cube or list of cubes to be saved
         filename (str):
             Filename to save input cube(s)
+
+    Raises:
+        warning if cubelist contains cubes of varying dimensions.
     """
     if isinstance(cubelist, iris.cube.Cube):
         cubelist = [cubelist]
 
+    chunksizes = None
+    xy_chunksizes = [None, None]
     for cube in cubelist:
         check_cube_not_float64(cube)
         _order_cell_methods(cube)
         _check_for_units(cube)
 
```

Comment:
Tomek wanted a comment to explain this, something like:
```suggestion
    # If all xy slices are the same shape, use this to determine the chunksize for the netCDF (eg. 1, 1, 970, 1042)
```